portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,20 @@
+html {
+    background: url(images/ogdrawnwood.png);
+    background-position: center;
+    /*background-size: 840px 840px;*/
```

Comment:
is there a reason this is commented out?

Reply:
I was testing out different sizes for the background image, but ended up not needing it. Will delete because it isn't used!